docs/contributor/draft/rate-limit-design.md

```diff
@@ -0,0 +1,80 @@
+# Rate Limit Proposal
+
+## Spec 
+
+| field                                            | type        | description                                                     | required                      |
+|--------------------------------------------------|-------------|-----------------------------------------------------------------|-------------------------------|
+| wokloadSelector                                  | object      | when skipped, it's applied for the whole namespace workloads, if empty and in istio-system ns, applies to all workloads in all namespaces | no                            |
```

Comment:
I'd reuse documentation from Istio.
```suggestion
| workloadSelector                                  | object      | If omitted, the rate limit will be applied to all workload instances in the same namespace as the RateLimit CR. If the RateLimit CR is present in the istio-system namespace, it will be applied to all workloads in any namespace. | no                            |
```